src/build/config/config.yaml

```diff
@@ -1536,7 +1536,11 @@ option:
     group: repo
     secure: true
     command: repo-type
-    depend: repo-gcs-bucket
+    depend:
+      option: repo-gcs-key-type
+      list:
+        - service
```

Comment:
Why does auto not appear in these lists?

Reply:
What this is saying is that `repo-gcs-key` is only valid (i.e. settable) when `key-type` is `service` or `token`, since only those types require the user to supply a key. When the key type is auto we get the key programmatically so letting the user set it would be misleading.

Reply:
Thanks.